qpc/insights/configure.py

```diff
@@ -0,0 +1,64 @@
+"""InsightsConfigureCommand is used to set target host and port for insights upload."""
+from logging import getLogger
+
+from qpc import insights, messages
+from qpc.clicommand import CliCommand
+from qpc.insights.utils import validate_host
+from qpc.source.utils import validate_port
+from qpc.translation import _
+from qpc.utils import write_insights_config
+
+log = getLogger("qpc")
+
+
+class InsightsConfigureCommand(CliCommand):
+    """Defines insights configure command.
+
+    This command is for configuring the target
+    host and port server for insights upload.
+    """
+
+    SUBCOMMAND = insights.SUBCOMMAND
+    ACTION = insights.CONFIG
+
+    def __init__(self, subparsers):
+        """Create command."""
+        super().__init__(
+            self.SUBCOMMAND,
+            self.ACTION,
+            subparsers.add_parser(self.ACTION),
+            None,
+            None,
+            [],
+        )
+        self.parser.add_argument(
+            "--host",
+            dest="host",
+            metavar="HOST",
+            default="console.redhat.com",
```

Comment:
[optional] if you decide to set the default values to constants you could consume them here in the default values